### PR TITLE
Updated to rxdart: ^0.27.1 & added null-safety

### PR DESCRIPTION
**Pull request by tkeithblack**
This pull request includes two changes. One is to update to rxdart: ^0.26.0, the other is addition of null-saftey.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -16,6 +16,5 @@ dev_dependencies:
 flutter:
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
-  flutter: ">=1.12.0"
-
+  sdk: ">=2.13.0 <3.0.0"
```
Comment (MaikuB) [inline pubspec.yaml]: Could these constraints be lowered to earliest version with null safety support?

**Review by MaikuB — CHANGES_REQUESTED** [review]
Thanks a lot for the PR. Besides the comment I left on the constraints. Would you able to help look at migrating the example app itself to use null safety?

**Comment by MaikuB** [comment]
Made the changes myself so all good, thanks :)

**Comment by tkeithblack** [comment]
Thanks for doing the merge. Sorry I didn't get back to you on migrating the example.

**Pull request merged**